### ISSUE TITLE
NTR: Abnormal Cystatin C level

### ISSUE THREAD

**Issue by callahantiff** [obophenotype/human-phenotype-ontology#5629]
**Preferred term label:**  
- Abnormal cystatin C level  
- Decreased cystatin C level  
- Elevated cystatin C level  

<br>

**Synonyms**
- **Abnormal cystatin C level**: Abnormal cystatin C   
- **Decreased cystatin C level**:  Low serum cystatin C; Low plasma cystatin C    
- **Elevated cystatin C level**: High serum cystatin C; High plasma cystatin C; Excess cystatin C    

<br>

**Definition (free text, please give PubMed ID)**
- **Abnormal cystatin C level**:  An abnormal concentration of cystatin C in serum or plasma 
- **Decreased cystatin C level**:  A decreased concentration in blood or serum of cystatin C    
- **Elevated cystatin C level**: An increased concentration in blood or serum of cystatin C       

[PMID:24570004](https://www.ncbi.nlm.nih.gov/pubmed/24570004)

<br>

**Parent term (use hpo.jax.org/app)**
Looking at the current hierarchy, perhaps _Abnormal cystatin C level_ could be added as a subclass of _Abnormal circulating protein level (HP_0010876)_.

<br>

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**  
**Low cystatin C Level**  
- Arthritis [PMID:21619490](https://www.ncbi.nlm.nih.gov/pubmed/21619490)  
- Cardiovascular disease [PMID:23849291](https://www.ncbi.nlm.nih.gov/pubmed/23849291)

**high cystatin C Level**   
- Acute kidney injury [PMID: 23814415](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3692142/)  
- Cardiovascular disease [PMID:23849291](https://www.ncbi.nlm.nih.gov/pubmed/23849291)  

<br>

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-8169-9049

**Comment by pnrobinson** [comment]
added as HP_0032996 etc.

**Comment by callahantiff** [comment]
Thanks @pnrobinson! Will be uploading the Loinc2HPO mappings for these terms soon.
